crates/control-interface/src/lib.rs

```diff
@@ -361,40 +361,31 @@ impl Client {
         }
     }
 
-    /// Puts a config item for the given entity (actor or provider) ID at the given key, replacing
-    /// any data that is already present. Data is stored as a Vec of bytes, so anything that can be
-    /// turned into a vec is accepted as a parameter for the value.
+    /// Puts a named config, replacing any data that is already present.
     ///
-    /// Key names must be valid NATS subject strings and not contain any `.` or `>` characters.
-    ///
-    /// NOTE: This operation is currently experimental and may change or be removed at any time
+    /// Config names must be valid NATS subject strings and not contain any `.` or `>` characters.
```

Comment:
What could we catch if we validated that it was a valid subject string? I believe the error case is getting a malformed sunject error when we try to request, but I'm unsure exactly what the user would see

Reply:
Yeah, I think rust errors are bit better now, but definitely a future improvement we should do here